site/app/handlers.js

```diff
@@ -29,41 +29,58 @@ const isTokenValid = (date) => {
   return CLOCK_CONTINGENCY < delta && delta < VALID_TOKEN_AGE
 }
 
+const isAuthorizedImageUploadUser = (user) => {
+  return !!user
```

Comment:
Possibly add an ONS Jira reference to a ticket to authorize based on group membership (may need to be created)

Reply:
Will do this - I have a list of tickets to make